app/src/main/res/values/strings_about.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
     <string name="about_activity_title">About</string>
 
-    <string name="about_description">Open source two-factor authentication for Android</string>
+    <string name="about_description">An open source two-factor authentication for Android</string>
```

Comment:
This should either be "An open source two-factor authentication app for Android"  or not start with An

Reply:
True! Though I feel "An open source......" sounds more nicer!